hardware/simulation/simulation.mk

```diff
@@ -7,6 +7,34 @@ ifeq ($(VCD),1)
 DEFINE+=$(defmacro)VCD
 endif
 
+ifeq ($(TEST1),1)	
```

Comment:
Only one test for now, remove multiple tests since we don't know what they are

Reply:
OK